src/SFA.DAS.ApplyService.Application/Apply/Roatp/RoatpPreambleQuestionBuilder.cs

```diff
@@ -161,6 +168,13 @@ public class ExperienceAndAccreditations
             public static string StartPage = "235";
         }
 
+        public class CriminalComplianceChecks
```

Comment:
Did we ever get to the bottom of why we have multiple classes in the same .cs file?

Reply:
I've suggested to Mark that we do a tech debt story to go through the codebase and split classes out, if that's the coding standards we're working to

Reply:
👍